or should we fix this in openjpeg?

#--- configure.ac
#+++ configure.ac
#@@ -116,7 +116,8 @@
#   AC_CHECK_LIB([openjpeg], [opj_cio_open],,
# 	       AC_MSG_ERROR("*** libopenjpeg library not found ***"))
#   AC_CHECK_HEADERS([openjpeg.h],,
#-		   AC_MSG_ERROR("*** libopenjpeg headers not found ***"))
#+		   AC_MSG_ERROR("*** libopenjpeg headers not found ***"),
#+		   [#include <stdbool.h>])
# elif test x$enable_libopenjpeg = xtry; then
#   AC_CHECK_LIB([openjpeg], [opj_cio_open],
#                [enable_libopenjpeg="yes"],
--- configure
+++ configure
@@ -17990,7 +17990,8 @@
 
 for ac_header in openjpeg.h
 do :
-  ac_fn_cxx_check_header_mongrel "$LINENO" "openjpeg.h" "ac_cv_header_openjpeg_h" "$ac_includes_default"
+  ac_fn_cxx_check_header_mongrel "$LINENO" "openjpeg.h" "ac_cv_header_openjpeg_h" "$ac_includes_default"'
+'"#include <stdbool.h>"
 if test "x$ac_cv_header_openjpeg_h" = x""yes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_OPENJPEG_H 1
